gyft.py

```diff
@@ -149,16 +149,8 @@ def main():
             generate = False
     
     if generate:
-        INPUT_FILENAME = args.input if args.input else "data.txt"
         OUTPUT_FILENAME = args.output if args.output else "timetable.ics"
 
-        # check if data.txt exists
-        if osp.exists(INPUT_FILENAME):
-            ow = input(f"Timetable file {INPUT_FILENAME} exists. Do you want to overwrite it? (y/n): ")
-            print()
-            if ow.lower() == 'n':
-                overwrite = False
-
         if overwrite:
```

Comment:
You can remove the `overwrite` flag altogether as that was just for overwriting `data.txt`, which is not present anymore